crates/context_aware_config/src/api/context/handlers.rs

```diff
@@ -526,3 +530,68 @@ async fn bulk_operations(
     })?;
     Ok(Json(response))
 }
+
+#[put("/priority/recompute")]
+async fn priority_recompute(
+    db_conn: DbConnection,
+) -> superposition::Result<HttpResponse> {
```

Comment:
The return type could have been `superposition::Result<Vec<PriorityRecomputeResponse>>`